apps/server/src/modules/files/entity/file.entity.ts

```diff
@@ -36,13 +36,13 @@ export interface FileEntityProps {
 @Entity({ collection: 'files' })
 @Index({ options: { 'permissions.refId': 1 } })
 export class FileEntity extends BaseEntityWithTimestamps {
-	@Property({ nullable: true })
+	@Property({ type: 'date', nullable: true })
 	deletedAt?: Date;
 
-	@Property()
+	@Property({ type: 'boolean' })
```

Comment:
was this also creating problems?

Reply:
Yes. I Only changed it if there were problems?

Reply:
based on what? Should we not make it safer and do this eveywhere where we have boolean props in entities?
examples: 
task entity private = true;
lesson entity hidden = false
column-board-target entity published = false